Add ransomware report to documentation (#1404)
guardicore/monkey: docs/content/reports/ransomware.md

---
title: "Ransomware report"
date: 2021-08-05T13:23:10+03:00
draft: false

> description: "Provides information about ransomware simulation on your network"

Reviewer: I'm thinking we should explicitly set the order of the report pages in the sidebar.
![image](https://user-images.githubusercontent.com/44770317/128539943-ac803f50-3603-4a73-b5f9-d866fe09c110.png)

Let's add `weight` here (and in the other report pages) to control the order. My suggestion would be: 
Security
Zero Trust
ATT&CK
Ransomware

---

{{% notice info %}}
Check out [the documentation for other reports available in the Infection Monkey]({{< ref "/reports" >}}).
{{% /notice %}}

> The Infection Monkey's **Ransomware Report** provides you with information and insight into ransomware simulation on your network.

Reviewer: How about something like this?

The Infection Monkey can be configured to [simulate a ransomware attack](../usage/scenarios/ransomware-simulation/) on your network, whose findings can be found in the **Ransomware Report**.

> ## Breach

Reviewer: Let's add a section here giving a list of the sections, just like the Security Report page:
![image](https://user-images.githubusercontent.com/44770317/128541765-c86093e6-646a-45db-ba03-f632e6d3604a.png)

> The breach section is showing report of the machines that the ransomware simulation started from, as well as the ip address and starting time.
> 

Reviewer: ```suggestion
This section shows details of the root of the ransomware simulation including the machines from which it started, their IP addresses, and the starting times.
```

![Breach](/images/usage/reports/ransomware_report_1_breach.png "Breach")

## Lateral Movement

> This section provides information about lateral movement phase of the attack in the network. That mean employing various
> techniques in order to exploit other systems in the network.

Reviewer: ```suggestion
This section provides information about how the Infection Monkey moved through the network by comprising and exploiting other systems.
```

[See real-world examples on Guardicore's blog.](https://www.guardicore.com/blog/stopping-ransomware-with-segmentation/)

![Lateral Movement](/images/usage/reports/ransomware_report_2_lateral_movement.png "Lateral Movement")

## Attack

> Attack section is how much and which files were successfully encryption on the compromised systems.

Reviewer: ```suggestion
This section shows the details of file encryption over your network. This includes the total number of files that were encrypted, the machines on which they were accessed, their paths, and the encryption algorithm used on each.
```
